ios/keyman/Keyman/Keyman/Classes/InfoViewController/InfoViewController.swift

```diff
@@ -42,6 +49,39 @@ class InfoViewController: UIViewController, UIWebViewDelegate {
     reloadKeymanHelp()
   }
 
+  private func updateButtons() {
+    backButton.isEnabled = webView.canGoBack
+    forwardButton.isEnabled = webView.canGoForward
+  }
+
+  func webViewDidStartLoad(_ webView: UIWebView) {
+    UIApplication.shared.isNetworkActivityIndicatorVisible = true
+    updateButtons()
+  }
+
+  func webViewDidFinishLoad(_ webView: UIWebView) {
+    UIApplication.shared.isNetworkActivityIndicatorVisible = false
+    updateButtons()
+  }
+
+  func webView(_ webView: UIWebView, didFailLoadWithError error: Error) {
+    UIApplication.shared.isNetworkActivityIndicatorVisible = false
+    updateButtons()
+    log.debug(error)
```

Comment:
Note to self:  either remove it or Sentry-log it.  I'm tempted to say "Sentry-log", but it is possible to go fully online via some of the links included within the bundle, so... that seems a bit risky.